base/not_null_test.cpp

```diff
@@ -262,5 +268,25 @@ TEST_F(NotNullTest, RValue) {
   EXPECT_EQ(5, *owner_const_int);
 }
 
+TEST_F(NotNullTest, TypeTraits) {
+  // not_null<foo*> is trivially copyable.
+  EXPECT_TRUE(std::is_trivially_copyable_v<not_null<int*>>);
```

Comment:
Change all of those to `static_assert` (cf. https://github.com/mockingbirdnest/Principia/blob/master/numerics/next_test.cpp which is tests constexpr code).

Reply:
Done.